microservices/webhooks/index.js

```diff
@@ -0,0 +1,54 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const app = express();
+const superagent = require("superagent");
+const port = parseInt(process.env.PORT) || 8080;
+const token = process.env.WEBHOOKS_MICROSERVICE_TOKEN || "123456";
```

Comment:
maaaaaybe just make it exit early (with error and exit code) if `WEBHOOKS_MICROSERVICE_TOKEN` can't be found? just in case it gets accidentally deleted or something, then before it gets noticed its a vulnerability that anyone can just post with token `123456`

Reply:
This is a really good point. Initially that was set to allow running in developer mode. But good idea.

I'll add a check to ensure that they exist, unless in development mode, where it can then append these.

Thanks a ton for the feedback!